### ISSUE TITLE
Reproducibility issue among Quarto and Jupyter notebook files

### ISSUE THREAD

**Issue by holiday-sean** [UBC-DSCI-310-2025W2/dsci-310-group-01-jet2holiday#22]
Because there was no `set.seed(310)` both in `03_train_test_split.R` and in `airbnb_price_analysis.ipynb`, there is a difference in the vif & correlation tables among the quarto report and jupyter notebook.

The following changes must be made:
- Add `set.seed(310)` in `airbnb_price_analysis.ipynb`
- Add `set.seed(310)` in `03_train_test_split.R`

I know that the figures and data files in the `results` folder will be invalid now (since they change every time you run the script). But after the Makefile implementation, we can just run the `clean` command and run the `make all` command (after implementing the seed fix), which should make the quarto and jupyter notebook contain the same data.
